### DISCUSSION TITLE
Anyone have better examples than the ones provided in docs?

### DISCUSSION THREAD

**Discussion by fayazara** [f5/unovis#446] [Q&A]
The docs are super confusing, I have no idea what data to pass, can someone help me with this?

Below example has zero context and all examples in docs are like this
![CleanShot 2024-09-17 at 23 19 09@2x](https://github.com/user-attachments/assets/9ee4134b-72bc-43da-b9e0-22f553bd5bf3)

**Comment by lee00678** [comment]
Hi @fayazara, we have a dedicated gallery page with lots of different examples. Maybe take a look at that and see if that helps? 
For example, here's a simple [donut chart](https://unovis.dev/gallery/view?collection=Circular%20Charts&title=Basic%20Donut%20Chart).

**Reply by fayazara** [reply]
Like I mentioned none of the examples show how the data is supposed yo oook like, thr example you showed also doesn't show the data.

**Reply by lee00678** [reply]
> Like I mentioned none of the examples show how the data is supposed yo oook like, thr example you showed also doesn't show the data.

All the examples on the gallery page come with a data tab, which showcases the format of data. Below is an example of the donut chart's data tab I linked earlier. 
![Screenshot 2024-09-17 at 12 11 08 PM](https://github.com/user-attachments/assets/6a0110e2-45b0-453e-a763-180b1302354f)

In the vue example you import it as `import { data, DataRecord } from './data'`
`./data` is the your data file.

**Comment by fayazara** [comment]
![CleanShot 2024-09-18 at 00 46 48@2x](https://github.com/user-attachments/assets/8fe98a75-3155-4f75-8f8e-b89164184d77)
Why can't I see this?? 😅

**Comment by fayazara** [comment]
Okay, its only available on the Gallery, not on the component reference, this is a bit confusing tbh

**Comment by fayazara** [comment]
Are there any examples for simple charts? Single bar chart, single line graph. 

Almost all examples on the Gallery are Grouped Bar, Multiline graph with gaps
![CleanShot 2024-09-18 at 00 49 27@2x](https://github.com/user-attachments/assets/3de8f540-06df-45fd-9561-d860f0663073)

**Comment by lee00678** [comment]
Hi @fayazara, we have a [simple line chart](https://unovis.dev/gallery/view?collection=Line%20Charts&title=Basic%20Line%20Chart) on the gallery page. As for simple bar chart, we have Grouped bar and stacked bar components. If you want to render a simple bar chart, just simply pass in an array of one Y values to a grouped bar component, and that should give you a simple bar chart. See an example [here](https://stackblitz.com/edit/qj97hr?file=src%2FApp.vue).
